src/app.js

```diff
@@ -12,7 +12,7 @@ import Help from './pages/help';
 import './app.css';
 
 const App = () => (
-  <BrowserRouter>
+  <BrowserRouter basename={process.env.PUBLIC_URL}>
```

Comment:
Does this mean we need to set the env variable `PUBLIC_URL` somewhere in the deployment?

Reply:
I think it's built into node.